### ISSUE TITLE
Support table split

### ISSUE THREAD

**Issue by fwang2002** [go-gorp/gorp#135]
I looked through the source code. The table name can not be got dynamically from the application layer when bindInsert, bindGet, etc are called. It is need when table split is used. For example, the "user" table's data is split into user1 and user2 which have the same schema.
It's also good to support custom sql.DB when the specified table is used

**Comment by GeertJohan** [comment]
I'm splitting your issue into two issues.

**Comment by GeertJohan** [comment]
Why do you have two tables with different names having the same schema? Is it maybe meant for some load balancing purposes? That would also explain the need for a custom `sql.DB`.

**Comment by fwang2002** [comment]
Yes. It's for load balancing purpose. Used when splitting tables.

**Comment by GeertJohan** [comment]
I'm not sure if we should add load balancing features to gorp. But lets think about this after v2 was released (see #270).
